ShareX/WorkerTask.cs

```diff
@@ -398,6 +398,17 @@ private void DoUploadJob()
 
                 if (!cancelUpload)
                 {
+                    if (Program.Settings.RemoveUploadedImageMetadata)
+                    {
+                        string[] supportedExtensions = { @".jpg", @".jpeg", @".png", @".tiff" };
```

Comment:
What about extensions that are provided via windows OS codecs that extends onto the WIC (Windows Imaging Component)? Should these be added to the supportedExtensions array too if found?